### PR TITLE
fix: #521 Modify the format according to the coding standards

### PR DESCRIPTION
**Pull request by KevinYang-github**
Modify the format according to the coding standards

### CODE REVIEW

File: app/models/strategyModel.js [hunk 1]
```diff
@@ -53,5 +53,8 @@ const StartegySchema = new mongoose.Schema(
   },
 );
 
+StartegySchema.statics.INVESTMENT_TYPE_REGULAR = '1';
```
Comment (Yingxue0323) [inline app/models/strategyModel.js:53]: why did you add  statics types in model? I thought we've got statusCode and strategyStateEnum already?

**Review by ucfyao — APPROVED** [review]

**Review by ucfyao — APPROVED** [review]
